src/components/EnterpriseApp/index.jsx

```diff
@@ -133,11 +133,6 @@ class EnterpriseApp extends React.Component {
                 <ProductTours />
                 <Sidebar
                   baseUrl={baseUrl}
-                  onWidthChange={(width) => {
-                    this.setState({
-                      sidebarWidth: width + defaultContentPadding,
-                    });
-                  }}
```

Comment:
Hm, so removing `onWidthChange` means the user can no longer use the hamburger menu icon in the header to persist the sidebar menu open. 

<img width="754" alt="image" src="https://user-images.githubusercontent.com/2828721/208150218-6aa9554e-4667-4554-ba04-78f9be1910de.png">

When the hamburger menu icon is clicked, it should treat the sidebar nav as an always-expanded menu with appropriate padding.

It might be worth checking what triggers the callback passed to `onWidthChange` in `Sidebar`. E.g., there is a `useEffect` with `onWidthChange` in its dependency array. But the callback function we're passing to `onWidthChange` in this component is not memoized so any re-render of `EnterpriseApp` would cause the callback function to have a new function reference, effectively triggering that `useEffect` in `Sidebar`.

I'd be curious if we abstracted this callback function out into a `useCallback` so the function reference persists on re-renders, and doesn't trigger that `useEffect` extra times.